ecosystem/sep-0010.md

```diff
@@ -188,6 +206,12 @@ Every other HTTP status code will be considered an error. For example:
 }
 ```
 
+## Validating the JWT
+
+Services that use the JWT as proof the user meets a threshold of control of a Stellar account need to validate the JWT. General [JWT best practices](#jwt-best-practices) should be followed for validating the claims within the header and payload. If the JWT contains threshold (`thr`) and weight (`wht`) claims the service should check that the threshold required by that service is met.
+
+Some versions of SEP-10 did not include the `thr` and `wht` and if the claims are not present the service can assume all thresholds were met.
```

Comment:
why do we want to assume all thresholds were met, vs no thresholds were met?

Reply:
Because that's what the SEP provides today. An implementer who is validating that the user has complete control of the account using the SEP as it currently exists is issuing JWTs without these claims, but the JWT is still claim to an account.

For an implementer who is not validating that the user has complete control of the account though this could be an issue. So maybe I should remove this statement all together and not recommend what an implementer should do. An implementer will usually be in control in the producer and consumer of a JWT, so I don't think we need to worry about cross-service compatibility.

I've removed the statement in e83f645.